app/services/book_service.rb

```diff
@@ -43,8 +39,10 @@ def self.remove_book(shelf_id, volume_id, auth_token)
 
   def self.conn(auth_token = nil)
     Faraday.new(url: 'https://books.googleapis.com') do |faraday|
-      faraday.params[:key] = ENV['BOOK_KEY']
       faraday.headers[:Authorization] = "Bearer #{auth_token}"
+      # faraday.params[:key] = ENV['BOOK_KEY']
+        # 8/2/21 - not sure why, but this breaks functionality in development.
+        # All calls work without it - to revisit if needed.
```

Comment:
There isn't any need for it in any call that uses authorization. Strange that the docs said they don't care if you send it as well, but I am so glad you guys figured this out. Massive roadblock you kicked down here!